docs/docs-network/operation/keystore/creating_keystores.md

```diff
@@ -28,13 +28,7 @@ Before creating keystores, ensure you have:
 First, install the Aztec CLI using the official installer:
 
 ```bash
-bash -i <(curl -s https://install.aztec.network)
-```
-
-Then install the correct version for the current network:
-
-```bash
-aztec-up 2.1.7
+bash -i <(curl -s https://install.aztec.network/2.1.9/)
```

Comment:
@sklppy88 is adding a MAINNET_TAG in this PR: https://github.com/AztecProtocol/aztec-packages/pull/19121

Esau, we could update this to be `bash -i <(curl -s https://install.aztec.network/#include_mainnet_version/)` once that other PR is merged, right?

Reply:
sometimes when we do releases we want to change certain version tags, but not others. if we do a devnet release, all of the mainnet tags should stay the same across the docs. if we do a mainnet release, all of the devnet tags should stay the same